samples/financial_transcripts/generate_benchmark_qa.ipynb

```diff
@@ -0,0 +1,562 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "Ticker = np.random.choice(df['Ticker'].unique())\n",
+    "Year = np.random.choice(df[df['Ticker']==Ticker]['Year'].unique())\n",
+    "Quarter = np.random.choice(df[(df['Ticker']==Ticker) & (df['Year']==Year)]['Quarter'].unique())\n",
+    "Id = np.random.choice(df[(df['Quarter']==Quarter) & (df['Year']==Year)]['Id'].unique())\n",
+    "Id2 = np.random.choice(df[(df['Quarter']==Quarter) & (df['Year']==Year) & (df['Id']!=Id)]['Id'].unique())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'MSFT'"
+      ]
+     },
+     "execution_count": 10,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "Ticker"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Generate Questions (using Azure OpenAI only)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 22,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "1. What was the percentage increase in server products and cloud services revenue for MSFT in Q1 FY23?\n",
+      "Answer: The server products and cloud services revenue increased by 22% in constant currency for MSFT in Q1 FY23.\n",
+      "\n",
+      "2. How much did Azure and other cloud services revenue grow in constant currency for MSFT in Q1 FY23?\n",
+      "Answer: Azure and other cloud services revenue grew 35% in constant currency for MSFT in Q1 FY23.\n",
+      "\n",
+      "3. Did Azure consumption growth meet expectations for MSFT in Q1 FY23?\n",
+      "Answer: No, Azure consumption growth was about a point lower than expected for MSFT in Q1 FY23.\n",
+      "\n",
+      "4. What was the impact on the enterprise mobility and security installed base for MSFT in Q1 FY23?\n",
+      "Answer: The enterprise mobility and security installed base grew by 18% to over 232 million seats for MSFT in Q1 FY23.\n",
+      "\n",
+      "5. How did the revenue for MSFT's on-premises server business perform in Q1 FY23?\n",
+      "Answer: The revenue was flat and increased 4% for MSFT's on-premises server business in Q1 FY23.\n",
+      "\n",
+      "6. What was the reason for the decrease in MSFT's margins in Q1 FY23?\n",
+      "Answer: MSFT elected not to throttle back OpEx despite the weakness on the Windows side, which caused margins to drift a little bit lower in Q1 FY23.\n",
+      "\n",
+      "7. Why did MSFT decide not to throttle back OpEx in Q1 FY23?\n",
+      "Answer: MSFT wanted to continue to invest where they were seeing substantial growth, and they believe it's important to continue investing in those areas.\n",
```

Comment:
that's not an answer to the point, rather a general high-level statement that Amy uses as an introduction into the real answer